intermediate_html_css/intermediate_css_concepts/advanced_selectors.md

```diff
@@ -86,6 +86,8 @@ Finally, if we want to select all of an element's siblings and not just the firs
 
 Just like the descendant combinator, these selectors don't have any special specificity rules - their specificity score will just be made up of their component parts.
 
+You can also check out [the docs](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors/Combinators) on this to find out more about combinators.
```

Comment:
```suggestion
This [MDN article on combinators](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors/Combinators) provides a good overview if you want to learn more about them.
```

I edited the sentence a bit to make the link more descriptive. What do you think of this wording?

Reply:
Yep! That works very well. I'll commit this.